{"level":"error","message":"[handleAbortError] AI response error; aborting request: {\"type\":\"no_user_key\"}","stack":"Error: {\"type\":\"no_user_key\"}\n    at getUserKey (/app/api/server/services/UserService.js:47:11)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getUserKeyValues (/app/api/server/services/UserService.js:68:20)\n    at async initializeClient (/app/api/server/services/Endpoints/openAI/initializeClient.js:41:18)\n    at async AskController (/app/api/server/controllers/AskController.js:54:24)\n    at async /app/api/server/routes/ask/openAI.js:24:5"}
{"code":"404","error":{"code":"404","message":"Resource not found"},"headers":{"apim-request-id":"e63a5bd8-16a4-4b0b-9a60-02a319bd54db","content-length":"56","content-type":"application/json","date":"Sun, 18 Aug 2024 09:15:52 GMT","strict-transport-security":"max-age=31536000; includeSubDomains; preload","x-content-type-options":"nosniff"},"level":"error","message":"[handleAbortError] AI response error; aborting request: 404 Resource not found","stack":"Error: 404 Resource not found\n    at APIError.generate (/app/api/node_modules/openai/error.js:54:20)\n    at OpenAI.makeStatusError (/app/api/node_modules/openai/core.js:263:33)\n    at OpenAI.makeRequest (/app/api/node_modules/openai/core.js:306:30)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async ChatCompletionStream._createChatCompletion (/app/api/node_modules/openai/lib/ChatCompletionStream.js:53:24)\n    at async ChatCompletionStream._runChatCompletion (/app/api/node_modules/openai/lib/AbstractChatCompletionRunner.js:314:16)","status":404}
{"level":"error","message":"There was an uncaught error: Model named \"Azure Models\" not found in configuration.","stack":"Error: Model named \"Azure Models\" not found in configuration.\n    at exports.mapModelToAzureConfig (/app/packages/data-provider/dist/index.js:1:80115)\n    at OpenAIClient.titleConvo (/app/api/app/clients/OpenAIClient.js:802:11)\n    at addTitle (/app/api/server/services/Endpoints/openAI/addTitle.js:24:30)\n    at AskController (/app/api/server/controllers/AskController.js:167:7)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async /app/api/server/routes/ask/openAI.js:24:5"}
{"code":"404","error":{"code":"404","message":"Resource not found"},"headers":{"apim-request-id":"7ff37020-a4c9-4435-af94-65b876e5c995","content-length":"56","content-type":"application/json","date":"Sun, 18 Aug 2024 09:19:44 GMT","strict-transport-security":"max-age=31536000; includeSubDomains; preload","x-content-type-options":"nosniff"},"level":"error","message":"[handleAbortError] AI response error; aborting request: 404 Resource not found","stack":"Error: 404 Resource not found\n    at APIError.generate (/app/api/node_modules/openai/error.js:54:20)\n    at OpenAI.makeStatusError (/app/api/node_modules/openai/core.js:263:33)\n    at OpenAI.makeRequest (/app/api/node_modules/openai/core.js:306:30)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async ChatCompletionStream._createChatCompletion (/app/api/node_modules/openai/lib/ChatCompletionStream.js:53:24)\n    at async ChatCompletionStream._runChatCompletion (/app/api/node_modules/openai/lib/AbstractChatCompletionRunner.js:314:16)","status":404}
{"code":"404","error":{"code":"404","message":"Resource not found"},"headers":{"apim-request-id":"b773059b-1397-49ac-92be-d4574ef58956","content-length":"56","content-type":"application/json","date":"Sun, 18 Aug 2024 09:19:53 GMT","strict-transport-security":"max-age=31536000; includeSubDomains; preload","x-content-type-options":"nosniff"},"level":"error","message":"[handleAbortError] AI response error; aborting request: 404 Resource not found","stack":"Error: 404 Resource not found\n    at APIError.generate (/app/api/node_modules/openai/error.js:54:20)\n    at OpenAI.makeStatusError (/app/api/node_modules/openai/core.js:263:33)\n    at OpenAI.makeRequest (/app/api/node_modules/openai/core.js:306:30)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async ChatCompletionStream._createChatCompletion (/app/api/node_modules/openai/lib/ChatCompletionStream.js:53:24)\n    at async ChatCompletionStream._runChatCompletion (/app/api/node_modules/openai/lib/AbstractChatCompletionRunner.js:314:16)","status":404}
{"code":"404","error":{"code":"404","message":"Resource not found"},"headers":{"apim-request-id":"bd89ffb3-f014-4271-8c35-cb1e23261232","content-length":"56","content-type":"application/json","date":"Sun, 18 Aug 2024 09:20:06 GMT","strict-transport-security":"max-age=31536000; includeSubDomains; preload","x-content-type-options":"nosniff"},"level":"error","message":"[handleAbortError] AI response error; aborting request: 404 Resource not found","stack":"Error: 404 Resource not found\n    at APIError.generate (/app/api/node_modules/openai/error.js:54:20)\n    at OpenAI.makeStatusError (/app/api/node_modules/openai/core.js:263:33)\n    at OpenAI.makeRequest (/app/api/node_modules/openai/core.js:306:30)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async ChatCompletionStream._createChatCompletion (/app/api/node_modules/openai/lib/ChatCompletionStream.js:53:24)\n    at async ChatCompletionStream._runChatCompletion (/app/api/node_modules/openai/lib/AbstractChatCompletionRunner.js:314:16)","status":404}
{"level":"error","message":"There was an uncaught error: Model named \"Azure Models\" not found in configuration.","stack":"Error: Model named \"Azure Models\" not found in configuration.\n    at exports.mapModelToAzureConfig (/app/packages/data-provider/dist/index.js:1:80115)\n    at OpenAIClient.titleConvo (/app/api/app/clients/OpenAIClient.js:802:11)\n    at addTitle (/app/api/server/services/Endpoints/openAI/addTitle.js:24:30)\n    at AskController (/app/api/server/controllers/AskController.js:167:7)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async /app/api/server/routes/ask/openAI.js:24:5"}
{"level":"error","message":"There was an uncaught error: Model named \"Azure Models\" not found in configuration.","stack":"Error: Model named \"Azure Models\" not found in configuration.\n    at exports.mapModelToAzureConfig (/app/packages/data-provider/dist/index.js:1:80115)\n    at OpenAIClient.titleConvo (/app/api/app/clients/OpenAIClient.js:802:11)\n    at addTitle (/app/api/server/services/Endpoints/openAI/addTitle.js:24:30)\n    at AskController (/app/api/server/controllers/AskController.js:167:7)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async /app/api/server/routes/ask/openAI.js:24:5"}
{"level":"error","message":"There was an uncaught error: Model named \"Azure Models\" not found in configuration.","stack":"Error: Model named \"Azure Models\" not found in configuration.\n    at exports.mapModelToAzureConfig (/app/packages/data-provider/dist/index.js:1:80115)\n    at OpenAIClient.titleConvo (/app/api/app/clients/OpenAIClient.js:802:11)\n    at addTitle (/app/api/server/services/Endpoints/openAI/addTitle.js:24:30)\n    at AskController (/app/api/server/controllers/AskController.js:167:7)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async /app/api/server/routes/ask/openAI.js:24:5"}
{"level":"error","message":"There was an uncaught error: Model named \"Azure Models\" not found in configuration.","stack":"Error: Model named \"Azure Models\" not found in configuration.\n    at exports.mapModelToAzureConfig (/app/packages/data-provider/dist/index.js:1:80115)\n    at OpenAIClient.titleConvo (/app/api/app/clients/OpenAIClient.js:802:11)\n    at addTitle (/app/api/server/services/Endpoints/openAI/addTitle.js:24:30)\n    at AskController (/app/api/server/controllers/AskController.js:167:7)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async /app/api/server/routes/ask/openAI.js:24:5"}
